### ISSUE TITLE
Is there a "in new window" for the various Vomnibar and LinkHints functions?

### ISSUE THREAD

**Issue by parogen** [gdh1995/vimium-c#1197]
Say for `Vomnibar.activateInNewTab`, is it possible to have `Vomnibar.activateInNewWindow` alternative? As well as for links, history, bookmarks?

Partially related is that when I use `moveTabToNewWindow`, the page is not focused and I need to click on the page to access vimium. While `openUrl`, `createTab reuse="new-window"`, as well as `Ctrl-N` (new window) with Vimium C NewTab Adapter all seem to focus on the page and allow me to use vimium immediately.

If yes, it would also be nice for the options page to be able to open in a new window as well.

**Comment by gdh1995** [comment]
Yes the two groups of commands doesn't support the `reuse` parameter. It's because they're implemented much earlier than the `reuse` feature.

For `LinkHints.*`, you may add `newtab="window"` or `newtab="last-window"` to ensure Vimium C opens a URL in a new window or another just-opened/visited window. While this means you may not use `Ctrl` and `Shift` to switch among different modes any more.

As for `Vomnibar`, sorry again there's no way to do so now. If you do want to open URL in a new window, you may create a custom search engine like:
```
nw: vimium://run/openUrl#reuse=new-window&#url=$S Open In New Window
```

And then: 1) open Vomnibar and type query words, 2) press ArrowDown to select an item, 3) move text caret to the start of input box and prepend "`nw `", 4) press Enter

**Comment by gdh1995** [comment]
> `moveTabToNewWindow`, the page is not focused and I need to click on the page to access vimium

As for this issue, it sounds like a bug.

Do you use Linux-based desktop systems and Firefox? If so, then this may be a long-standing issue that
- with some desktop window managers on Linux and Firefox, `to create a new window` may make an existing tab lose focus and there's no perfect way to fix it.

**Comment by parogen** [comment]
> Do you use Linux-based desktop systems and Firefox?

I am on Windows 10 and Firefox. As mentioned, it's specifically with that function only. Other ways of opening a new window or switching to another browser window (like move tab between existing windows) through Vimium do not have this issue for me.

The new parameter works for the non-Vomnibar Linkhints functions. I think incorporating a search keyword just for new window is a bit much for me. There is already an existing way of opening the new tab, then using `moveTabToNewWindow` afterwards (though ironically I have the mentioned issue), if more steps is not a burden.
